packages/webpack/src/ember-webpack.ts

```diff
@@ -184,10 +184,10 @@ const Webpack: PackagerConstructor<Options> = class Webpack implements Packager
         ],
       },
       output: {
-        path: join(this.outputPath, 'assets'),
-        filename: `chunk.[chunkhash].js`,
-        chunkFilename: `chunk.[chunkhash].js`,
-        publicPath: publicAssetURL + 'assets/',
+        path: join(this.outputPath),
+        filename: `assets/chunk.[chunkhash].js`,
```

Comment:
do we have existing tests to assert these files are created in this folder?

Reply:
Tests like[these](https://github.com/embroider-build/embroider/blob/9271aa877d70e4c1bf214fc4e34321ffeeb8d1ae/packages/compat/tests/stage2.test.ts#L597-L611) are directly inspecting the output files.

And the various tests/scenarios are ensuring that these files actually execute.